mininet/node.py

```diff
@@ -746,6 +749,15 @@ def __init__(
             dns=self.dns,
         )
 
+        # TODO: remove docker before creating it -- Jinwoo Kim
+        container_list = self.dcli.containers(all=True)
+
+        for container in container_list:
+            for container_name in container["Names"]:
```

Comment:
More robust version:
```suggestion
            for container_name in container.get("Names", []):
```